rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-galera.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Confirm wsrep_cluster_size
+  command: "mysql -e \"show status like 'wsrep_cluster_size'\""
+  register: mysql_cluster_size
+  failed_when: "'wsrep_cluster_size\t{{ groups['galera_all']|length }}' not in mysql_cluster_size.stdout"
+  changed_when: mysql_cluster_size.rc != 0
+
+- name: Confirm wsrep_cluster_status
+  command: "mysql -e \"show status like 'wsrep_cluster_status%'\""
+  register: mysql_cluster_status
+  failed_when: "'wsrep_cluster_status\tPrimary' not in mysql_cluster_status.stdout"
```

Comment:
I can prune the output of the command above so I'm comparing only a one word, and not anything else.